web/next.config.js

```diff
@@ -10,6 +10,8 @@ const withPWA = require('next-pwa')({
 
 /** @type {import('next').NextConfig} */
 const nextConfig = {
+  output: 'export',
```

Comment:
Let's set the output directory to the `dist` folder using `distDir: 'dist',`
```suggestion
  output: 'export',
  distDir: 'dist',
```

Reply:
I'm going to go ahead to apply this suggestion